pkg/cli/demo_locality_test.go

```diff
@@ -17,14 +17,19 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/cli/democluster"
 	"github.com/cockroachdb/cockroach/pkg/security/securityassets"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 )
 
-var _ = skipExample_demo_locality
-
-func skipExample_demo_locality() {
+func Example_demo_locality() {
```

Comment:
not familiar with this test. but looks like this is not a test or used anywhere. could you educate me how does this work?

Reply:
it's actually part of the go language. (something i didn't realize until i also saw these tests and were confused how they ran)

https://go.dev/blog/examples